homework-submission/week1-project/src/App.js

```diff
@@ -2,34 +2,61 @@ import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
 import Description from './Components/todoComponents.js'
-
+import TodoForm from './Components/addTodoForm'
+import JSONList from './JSONList'
 
 class App extends Component {
   
   constructor(props){
     super(props);
-    this.state = {list: props.JSONList};
+    this.state = {list: JSONList};
```

Comment:
I'd name `list` more explicit. e.t. `todoList` or `todos`. More importantly, use a name that you want to use elsewhere. 

What I mean is that you assign `JSONList` to list in the state. And then in the rest of your file, you do the opposite! e.t. line 16, 23, and 43!